### DISCUSSION TITLE
购买后没有给进入QQ群的方式说明

### DISCUSSION THREAD

**Discussion by Lorikein12138** [natfrp/wiki#231] [使用问题]
购买VIP后，提示可以通过密钥加入QQ群，但是我没有收到密钥以及QQ群的群号

**Comment by fakeboboliu** [comment]
关于密钥的问题，此问题是如何使用我们服务的基石：如果还没有开始使用我们的服务，请点一下用户面板那个到处飘的弹窗。

关于QQ群，我们的QQ群并不提供支持服务，如果您仍希望进入，请打开 [用户信息](https://www.natfrp.com/user/profile) 看右下角的“账户信息”部分。
